### DISCUSSION TITLE
hybridclr支持CSharpCodeProvider吗

### DISCUSSION THREAD

**Discussion by liamaurelian** [focus-creative-games/hybridclr#135] [Q&A]
hybridclr是可以解释执行的，希望在游戏里面增加可以运行时解析代码字符串，直接修改游戏里面的逻辑的功能，类似xlua这样的脚本语言，可以让游戏更加灵活。
查了下csharp有个CSharpCodeProvider库可以动态编译，不知道hybridclr是否支持

**Comment by pirunxi** [comment]
如果能编译为单独dll的话，是支持的
